distribution/docs/src/main/resources/_contents/_apis/application-service-api-contents.adoc

```diff
@@ -1,21 +1,19 @@
 
```

Comment:
I noticed that the `Application Service API` section title was repeated in the Documentation.